playbooks/roles/openvpn/defaults/main.yml

```diff
@@ -13,6 +13,9 @@ openvpn_clients:
 
 openvpn_port: "636"
 openvpn_port_udp: "8757"
+openvpn_port_sslh: "443"
 openvpn_key_size: "2048"
-openvpn_cipher: "AES-256-CBC"
+openvpn_cipher: "AES-256-GCM"
+openvpn_ncp_cipher: "AES-256-GCM:AES-256-CBC"
 openvpn_auth_digest: "SHA256"
+openvpn_connect_timeout: 15 #seconds
```

Comment:
I feel that 5 seconds might be a bit too low; based on experience when I used to live in Oman connectivity can be shady at times, and tunnels would take a bit longer than anticipated to establish.

15 seconds might also be a bit much in retrospect, so I dialed it down to 10. I feel that it's a reasonable compromise.